src/test/obj_ulog_size/obj_ulog_size.c

```diff
@@ -882,6 +882,69 @@ do_log_snapshots_max_size(PMEMobjpool *pop)
 	FREE(sizes);
 }
 
+/*
+ * do_ctl_snapshots_cache_size -- test setting the cache size
+ * transaction parameter for the snapshot cache
+ */
+static void
+do_ctl_snapshots_cache_size(PMEMobjpool *pop)
+{
+	UT_OUT("do_ctl_snapshots_cache_size");
+
+	long long cache_size = 0x7FFFFBB4;
+
+	/* cache size in the range- it should pass */
+	int ret = pmemobj_ctl_set(pop, "tx.cache.size",
+					&cache_size);
+	UT_ASSERTeq(ret, 0);
+
+	/* change value of cache size to make sure ctl_get works correctly */
+	cache_size = 0;
+	ret = pmemobj_ctl_get(pop, "tx.cache.size",
+					&cache_size);
+	UT_ASSERTeq(ret, 0);
+	UT_ASSERTeq(cache_size, 0x7FFFFBB4);
+
+	cache_size = (ssize_t)PMEMOBJ_MAX_ALLOC_SIZE - 1;
```

Comment:
use just `PMEMOBJ_MAX_ALLOC_SIZE`, it will be better test than `PMEMOBJ_MAX_ALLOC_SIZE -1 `